SPECS/systemd/CVE-2022-3821.patch

```diff
@@ -0,0 +1,24 @@
+From 8d2d0895229cfbe39c1c5c16e61e426812a72e8b Mon Sep 17 00:00:00 2001
+From: Yu Watanabe <watanabe.yu+github@gmail.com>
+Date: Thu, 7 Jul 2022 18:27:02 +0900
+Subject: [PATCH] time-util: fix buffer-over-run
+
+Fixes #23928.
+---
+ src/basic/time-util.c     | 2 +-
+ src/test/test-time-util.c | 5 +++++
```

Comment:
Note: I removed this test code change which is not included with our version of the code.

Reply:
Patch source: https://github.com/systemd/systemd/commit/8d2d0895229cfbe39c1c5c16e61e426812a72e8b.patch